text/0000-add-visual-pack-build.md

```diff
@@ -0,0 +1,80 @@
+# Meta
+[meta]: #meta
+- Name: Add Visual Pack Build
+- Start Date: 2020-05-14
+- Author(s): aemengo
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: https://github.com/buildpacks/rfcs/pull/152
+
+# Summary
+[summary]: #summary
+
+Pack should provide a visual representation of the image build process. Please see [mockup](https://storage.googleapis.com/artifacts.cf-garden-windows-dev.appspot.com/pack-build-visual.mp4) for concept.
+
+# Definitions
+[definitions]: #definitions
+
+* [Pack](https://github.com/buildpacks/pack) - A tool within the buildpacks ecosystem for converting code into runnable images, in a local environment.
+
+# Motivation
+[motivation]: #motivation
+
+The **casual application developer** has trouble seeing the value proposition of buildpacks. They also encounter a high learning curve when using our tooling. We've seen evidence of this through user interviews and other channels.
+
+For most of these developers, pack is their introduction to the buildpacks ecosystem. By providing a visual representation of the build process, they will be introduced to various buildpack concepts, and their relevance to the build process. 
+
+# What it is
+[what-it-is]: #what-it-is
+
+When a casual application developer runs `pack build [--interactive/-i]` on the command-line in the working directory of their application, they are presented with a terminal UI that depicts the build process. The command will depict the build process as four stages.
```

Comment:
The current proposal is dynamic as it's streaming and changing, but it's not terribly "interactive". Is there something else we could re-name this to? If I was a new developer I would expect using an interactive flag might drop me off in some kind of a stateful "wizard" flow or allow me to change some settings and observe their effects in real time. As is, it's a different streaming output.

Reply:
Thanks for the feedback. I agree it's not "interactive", but the goal is that it eventually would be. However the scope of true interactivity proved to be very large, even for an RFC, so this is supposed to be a first pass.

See here for the original vision: https://github.com/buildpacks/rfcs/pull/152

Reply:
Gotcha, is that the eventual goal and this is a stepping stone then? That makes sense as a path forward.

Reply:
> ...this is a stepping stone then?

Yep, absolutely!